waterloop-site/src/pages/Flock.tsx

```diff
@@ -3,11 +3,9 @@ import Hero from "components/Hero/General"
 import SimpleFeature from "components/SimpleFeatures/SimpleFeature"
 import TextWithImageData from "../static/copy/TextWithImage.json"
 
-var text = "";
 var image = "";
 
-TextWithImageData.map((detail) => {
-    text = detail.text;
+TextWithImageData.forEach((detail) => {
```

Comment:
Though these lines technically do the same thing, using map in this case goes against the whole point of the map function. The map function should be used to return a new array of objects which are a transformation of the previous array's objects. The forEach function should be used when you wish to apply a side effect to each element in an array. This is still not the best way of doing this, but I don't context of this component to be able to write it better